endpoints/bahmni/input-validation.json

```diff
@@ -1,553 +1,203 @@
+                "nullable": true
+              },
+              "address4": {
+                "type": "string",
+                "nullable": true
+              },
+              "address5": {
+                "type": "string",
+                "nullable": true
+              },
+              "address6": {
+                "type": "string",
+                "nullable": true
+              },
+              "address7": {
+                "type": "string",
+                "nullable": true
+              },
+              "address8": {
+                "type": "string",
+                "nullable": true
+              },
+              "address9": {
+                "type": "string",
+                "nullable": true
+              },
+              "address10": {
+                "type": "string",
+                "nullable": true
+              },
+              "address11": {
+                "type": "string",
+                "nullable": true
+              },
+              "address12": {
+                "type": "string",
+                "nullable": true
+              },
+              "address13": {
+                "type": "string",
+                "nullable": true
+              },
+              "address14": {
+                "type": "string",
+                "nullable": true
+              },
+              "address15": {
+                "type": "string",
+                "nullable": true
               }
             }
           }
-        }
-      },
-      "dateCreated": {
-        "type": "timestamp"
-      },
-      "changedBy": {
-        "type": "string"
-      },
-      "dateChanged": {
-        "type": "timestamp"
-      }
-    },
-    "deathdateEstimated": {
-      "type": "boolean",
-      "default": false
-    },
-    "birthtime": {
-      "type": "timestamp"
-    },
-    "links": {
-      "type": "array",
-      "element": {
-        "rel": {
-          "type": "string"
-        },
-        "uri": {
-          "type": "string"
-        }
-      }
-    },
-    "resourceVersion": {
-      "type": "string"
-    }
-  },
-  "voided": {
-    "type": "boolean"
-  },
-  "auditInfo": {
-    "type": "object",
-    "element": {
-      "creator": {
-        "uuid": {
-          "type": "string"
         },
-        "display": {
+        "gender": {
           "type": "string"
-        },
-        "links": {
-          "type": "array",
-          "element": {
-            "rel": {
-              "type": "string"
-            },
-            "uri": {
-              "type": "string"
-            }
-          }
         }
       }
-    },
-    "dateCreated": {
-      "type": "timestamp"
-    },
-    "changedBy": {
-      "type": "string"
-    },
-    "dateChanged": {
-      "type": "string"
-    }
-  },
-  "links": {
-    "type": "array",
-    "rel": {
-      "type": "string"
-    },
-    "uri": {
-      "type": "string"
     }
-  },
-  "resourceVersion": {
-    "type": "string",
-    "required": true
   }
-}
+}
```

Comment:
@bradsawadye Add newline at end of file pls

Reply:
fixed in https://github.com/jembi/openhim-mediator-mapping/pull/21/commits/e5b0206d05fd36b2b39f7140a36b0715f59c2b4d